packages/core/src/blockchainApiConnection/BlockchainApiConnection.ts

```diff
@@ -31,7 +29,7 @@ export const CUSTOM_TYPES: RegistryTypes = {
 }
 
 export async function buildConnection(
-  host: string = DEFAULT_WS_ADDRESS
+  host: string | undefined = get('address')
```

Comment:
What is the use case for calling `buildConnection(undefined)`?
If there is none
```suggestion
  host: string = get('address')
```
should do the trick.

Reply:
Ohh yes this is an artifact from before, is removed

Reply:
just thinking: maybe it's easier to read if we import `get` as `ConfigService.get` ?

Reply:
For me this wouldn't change comprehension, since we already import get from ConfigService, this is unnecessary IMO.